### ISSUE TITLE
hide epoll errors if not using debug flags in ops

### ISSUE THREAD

**Issue by eyberg** [nanovms/nanos#522]
by default debugging output should be hidden in ops - I think this is another one we need to hide

```
➜  t  ops load node_v11.15.0 -a hi.js
Downloading.. /Users/eyberg/.ops/packages/node_v11.15.0.tar.gz
 15.63 MiB / 15.63 MiB [=============================================================================================================================================================] 100.00% 3.50 MiB/s 4s
Extracting /Users/eyberg/.ops/packages/node_v11.15.0.tar.gz...
Downloading.. .staging/mkfs
 168.34 KiB / 168.34 KiB [===========================================================================================================================================================] 100.00% 3.60 MiB/s 0s
Downloading.. .staging/boot.img
 23.50 KiB / 23.50 KiB [=============================================================================================================================================================] 100.00% 3.69 MiB/s 0s
Downloading.. .staging/stage3.img
 1.41 MiB / 1.41 MiB [===============================================================================================================================================================] 100.00% 3.66 MiB/s 0s
[node hi.js]
booting image ...
assigned: 10.0.2.15
hi
epoll_ctl error: delete for unregistered fd 1
epoll_ctl error: delete for unregistered fd 2
exit_group
exit status 1
```

**Comment by eyberg** [comment]
well not seeing the epoll msgs but these are appearing and they shouldn't be in non-debug mode

```
ops load node_v11.5.0 -p 8083 -f -n -a hi.js
Downloading.. https://storage.googleapis.com/nanos/release/nightly/nanos-nightly-darwin.tar.gz
 579.69 KiB / 579.69 KiB [=========================================================================================================================================================] 100.00% 422.72 KiB/s 1s
[node hi.js]
booting /Users/eyberg/.ops/images/node.img ...
assigned: 10.0.2.15
Server running at http://127.0.0.1:8083/
error_message error: fd 19: unknown error -15
error_message error: fd 19: unknown error -15
error_message error: fd 19: unknown error -15
error_message error: fd 19: unknown error -15
error_message error: fd 19: unknown error -15
error_message error: fd 19: unknown error -15
error_message error: fd 19: unknown error -15
error_message error: fd 19: unknown error -15
error_message error: fd 19: unknown error -15
error_message error: fd 19: unknown error -15
error_message error: fd 19: unknown error -15
error_message error: fd 19: unknown error -15
error_message error: fd 19: unknown error -15
error_message error: fd 19: unknown error -15
error_message error: fd 19: unknown error -15
error_message error: fd 19: unknown error -15
error_message error: fd 19: unknown error -15
```

https://github.com/nanovms/nanos/blob/fe1af1eecd4587c300cc73d37a71714ec937553e/src/net/netsyscall.c#L213

**Comment by eyberg** [comment]
i think this is old/can be closed

**Comment by francescolavra** [comment]
Fixed by #1010, closing.
